### ISSUE TITLE
Alpine dockerfile example

### ISSUE THREAD

**Issue by shavo007** [DataDog/dd-trace-dotnet#561]
**Is your feature request related to a problem? Please describe.**
No problem. just wanted to showcase my setup if someone finds it useful

**Dockerfile**

```bash

FROM mcr.microsoft.com/dotnet/core/sdk:2.2.401-alpine3.9 AS build-env
WORKDIR /app/src
COPY src/ /app/src/
RUN dotnet publish -o temp -c Release

FROM mcr.microsoft.com/dotnet/core/aspnet:2.2.6-alpine3.9
ENV DD_APM_VERSION=1.9.0
RUN apk --no-cache update \
  && apk add bash make python py-pip libc6-compat gcompat curl
RUN pip install pyyaml
COPY Makefile /app/src/
COPY ./values/*.yaml /app/src/values/
COPY script/parseEnv.py /app/src/script/
COPY --from=build-env app/src/temp /app/src
#Install Datadog APM package
RUN  mkdir -p /opt/datadog
RUN curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v${DD_APM_VERSION}/datadog-dotnet-apm-${DD_APM_VERSION}.tar.gz \
  |  tar xzf - -C /opt/datadog
#Setup Datadog APM
ENV CORECLR_ENABLE_PROFILING=1
ENV CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
ENV CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
ENV DD_INTEGRATIONS=/opt/datadog/integrations.json
ENV DD_DOTNET_TRACER_HOME=/opt/datadog
WORKDIR /app/src
CMD ["dotnet", "app.dll"]
```

**app.csproj**

```
 <PackageReference Include="Datadog.Trace.ClrProfiler.Managed" Version="1.9.0" />

```

in startup class

```
  private static void ConfigureDataDogAPM()
    {
      var settings = TracerSettings.FromDefaultSources();
      string appName = Environment.GetEnvironmentVariable("APP_NAME");
      settings.ServiceName = appName;
      settings.AgentUri = new Uri("http://localhost:8126/");
      // disable the AdoNet integration
      settings.Integrations["AdoNet"].Enabled = false;
settings.Environment = Environment.GetEnvironmentVariable("ENVIRONMENT");

      // create a new Tracer using these settings
      var tracer = new Tracer(settings);
      // set the global tracer
      Tracer.Instance = tracer;

    }
```

```

**Comment by lucaspimentel** [comment]
Thanks, @shavo007! This will certainly help others setting up the Tracer on Alpine 3.9.

One bit of feedback: Since v1.8.0, if the `DD_DOTNET_TRACER_HOME` environment is set, the `Datadog.Trace.ClrProfiler.Managed` NuGet package is no longer required for .NET Core users, so feel free to remove the `<PackageReference>` from the `csproj` project file.

**Comment by shavo007** [comment]
awesome, thanks @lucaspimentel . 

from reading the doc https://docs.datadoghq.com/tracing/setup/dotnet/?tab=netcoreonlinux
it states "Install the .NET Tracer on the host using the using one of the packages available from the dd-trace-dotnet releases page."

If i do not need the package how i configure in my startup class?

**Comment by lucaspimentel** [comment]
To access the `Tracer` class for configuration or to use manual instrumentation, you only need the core `Datadog.Trace` package.

**Comment by shavo007** [comment]
cool, so just to confirm i still need a nuget package. But only `Datadog.Trace` for setting my env config and not `Datadog.Trace.ClrProfiler.Managed`?

**Comment by colin-higgins** [comment]
@shavo007 That is correct.

**Comment by colin-higgins** [comment]
Hello @shavo007, we now officially support automatic instrumentation with the new package (datadog-dotnet-apm-1.13.0-musl.tar.gz) available in the 1.13 release (https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.13.0)
